Athena's Wrath/Athena's Wrath/main.swift

```diff
@@ -0,0 +1,355 @@
+//
+//  main.swift
+//  Athena's Wrath
+//
+//  Created by Maitree Bain on 10/4/19.
+//  Copyright © 2019 Maitree Bain. All rights reserved.
+//
+
+import Foundation
+
+print("""
+You are Athena: Goddess of Wisdom and War. You sit atop Mount Olympus, along with powerful gods like Zeus, Poseidon and Hades and others. On an arbritrary cloudy day, the monsters attacked. Mount Olympus is being attacked by all of the monsters that were cursed by the Gods. Collect your weapons and start the fight!
+""")
+sleep(1)
+print("Type Start to start the game")
+
+var start = readLine()?.lowercased()
+if start == "start" {
+    print("A wild Minotaur appears!")
+    sleep(1)
+    print("Choose a weapon: Spear or Sword ")
+} else {
+    print("Please type in Start or exit the game.")
+}
+var run = true
+var weapon = readLine()?.lowercased()
+
+while run == true {
+    if weapon == "spear" {
+    print("You have chosen a: Spear!")
+        break
+} else if weapon == "sword" {
+    print("You have chosen a: Sword!")
+        break
+} else{
+    print("Please choose a weapon!")
```

Comment:
This line of code goes into effect if the user doesn't select "start" from the beginning but once it hits, it doesn't give the user weapon options to select from 🔢